examples_tests/ComputePointCloud/rasterizer.comp

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2018-2020 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+#version 450
+
+#pragma shader_stage(compute)
+
+layout (local_size_x = 256, local_size_y = 1) in;
+
+layout (push_constant) uniform pushConstants
+{
+	uvec2 imgSize;
+	uint pointCount;
+	mat4 mvp;
+} u_pushConstants;
+
+layout (set = 0, binding = 0, r32ui) uniform coherent uimage2D outImage;
+layout (set = 0, binding = 1, std430) readonly buffer pointCloud {
+	vec3 position[];
```

Comment:
`vec3` has a 16byte alignment, either:
- pad
- change format to PoT bitcount
- make a `struct{float x,y,z;};` and ue that instead of `vec3`